frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -268,35 +275,45 @@ export const ProjectAccessAssign = ({
             optionUser = option.entity as IUser;
         }
         return (
-            <li {...props}>
-                <Checkbox
-                    icon={<CheckBoxOutlineBlankIcon fontSize="small" />}
-                    checkedIcon={<CheckBoxIcon fontSize="small" />}
-                    style={{ marginRight: 8 }}
-                    checked={selected}
-                />
-                <ConditionallyRender
-                    condition={option.type === ENTITY_TYPE.GROUP}
-                    show={
-                        <StyledGroupOption>
-                            <span>{optionGroup?.name}</span>
-                            <span>{optionGroup?.userCount} users</span>
-                        </StyledGroupOption>
-                    }
-                    elseShow={
-                        <StyledUserOption>
-                            <span>
-                                {optionUser?.name || optionUser?.username}
-                            </span>
-                            <span>
-                                {optionUser?.name && optionUser?.username
-                                    ? optionUser?.username
-                                    : optionUser?.email}
-                            </span>
-                        </StyledUserOption>
-                    }
-                />
-            </li>
+            <Tooltip title={createRootGroupWarning(optionGroup)}>
```

Comment:
That's a pretty smart approach 👍

Reply:
You're just saying that 'cos you helped me come up with it :P